Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,60 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl/-/archive/v0.2/jpeg-xl-v0.2.tar.bz2"
+  sha256 "f0933c796f95ee905efa7a677367c0d57678b9587c2e967ea30d72e9405cca72"
+  license "Apache-2.0"
+
+  depends_on "clang-format" => :build
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
```

Comment:
Just following the [build](https://gitlab.com/wg1/jpeg-xl/-/blob/master/README.OSX.md) instructions.  LLVM isn't my forte so I'm happy to change it to follow your suggestion.

Reply:
I would assume coreutils is used from our CI/CD scripts ("ci.sh" bash script calls a bunch of other coreutils binaries, sometimes with flags that might not be available or compatible with other platforms). Since you are calling cmake directly I don't expect this to be used. The build instructions there were written by and for jxl devs documenting what needs to be installed to run all the other developer workflows besides just building the tool (like run benchmarks, format your patches, etc).